kubernetes/jobs/padigitale-report-generator/cronjob.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  labels:
+    app: {{SERVICE_NAME}}
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+spec:
+  schedule: "0 8 * * *"
```

Comment:
just curiosity, is it ok to start the job at 9AM/10AM CET?

Reply:
it's the same schedule that had before 🤷